src/fides/api/graph/traversal.py

```diff
@@ -114,6 +114,23 @@ def __init__(
                 )
             )
 
+        # Ensure manual_task collections execute right after ROOT
+        from fides.api.task.manual.manual_task_utils import ManualTaskAddress
```

Comment:
Manually linking the manual input nodes to root so that they are considered reachable